TKeazirian.HTTPServer/Server/Router.cs

```diff
@@ -19,9 +19,17 @@ public Response Route(Request request)
             IsHttpMethodAllowed(request))
         {
             var handler = _routesConfig.Routes[request.GetRequestPath()];
-            return request.GetRequestMethod() == "HEAD"
-                ? HeadRequest(handler, request)
-                : handler.HandleResponse(request);
+            if (request.GetRequestMethod() == "HEAD")
+            {
+                return HeadRequest(handler, request);
+            }
+
+            if (request.GetRequestMethod() == "OPTIONS" && request.GetRequestPath() != "/method_options2")
+            {
+                return OptionsRequest(handler, request);
+            }
```

Comment:
you have a hard-coded route here in the `!= "/method_options2"`.

Reply:
removed!